### ISSUE TITLE
CMS Design: Address CMS collab cycle notes, polish "Magic paragraph" UI design for Prod

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13755]
## Description
We need to provide a UX artifact for the Magic headers, to feed into the CMS Collab Cycle Design Review.

[Link to Magic Header Figma document](https://www.figma.com/file/0gH9R0yAadciGcy73vpsgX/Magic-Headers-and-Paragraphs?type=design&node-id=0%3A1&t=DaDnO1eSTdg5BMaH-1)

### User story

**AS A** PM/PO
**I WANT** the PwH (paragraph with header) UI in the VA Benefits Taxonomy UI to meet design and a11y standards
**SO THAT** we can release the feature to production with confidence it will meet Editor needs.

## Acceptance criteria

- [x] Jordan will clarify with Blake the nature of what kind of artifact he needs for review, Figma comp vs. Mural for annotating the prototype vs. prototype itself
- [ ] Design review prep: Documented UX design and functionality in the artifact per CMS Collab Cycle's requirements
  - [x] Confirm with CMS team where this should live – in PW > Benefits Taxonomy or central CMS location since this functionality will be added to other taxonomies/content types
- [ ] Design review follow up: CMS Collab Cycle Design review has been held and changes incorporated 
- [ ] A11y approved design artifact
- [ ] Engineering confirmed feasibility of design

**Comment by wesrowe** [comment]
Pending: it's possible that CMS CC will not require Figma at all? TBD.

**Comment by thejordanwood** [comment]
Updates from my meeting with Blake on 6/8:

- I created a [Figma document](https://www.figma.com/file/0gH9R0yAadciGcy73vpsgX/Untitled?type=design&node-id=0%3A1&t=nQixfBFI9iK5Ng6u-1) and added comments about design changes that Blake and I would like to see. There are a couple of blockers here that center around the heading levels interfering with accessibility.
- I set up a quick sync on 6/9 to go through some of these recommendations with Christia.
- I’m going to create a Figma mockup based on these recommendations and whatever comes out of our sync. We can look it over at our CMS design review meeting on Wednesday 6/14, along with whatever changes Christia makes to the prototype.

**Comment by thejordanwood** [comment]
Updates from my meeting with Christia on 6/9:

-  Christia was already working on a way to solve the heading level accessibility issues (yay!) and I believe the solve for now will be to give an error message after the save. She's going to look into finding a way to grey out or eliminate headings that shouldn't be used from the dropdown, but isn't sure if that's doable.
- The biggest change we decided to do is get rid of the "Add a paragraph with heading container" button, which will clear up some confusion and solve a couple of design problems that Blake and I identified yesterday. Users will still be able to add one paragraph container and add headings under the one container.
- Next steps for me are to create a Figma mockup that shows these changes and take a pass at editing the help text
